### PR TITLE
[SYCL] Disable SYCL/Basic/barrier_order.cpp on CUDA

### PR DESCRIPTION
**Pull request by aelovikov-intel**
See https://github.com/intel/llvm/issues/7330.

### CODE REVIEW

File: SYCL/Basic/barrier_order.cpp [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
 // Temporarily disabled on HIP and L0 due to sporadic failures.
```
Comment (sergey-semenov) [inline SYCL/Basic/barrier_order.cpp]: Could you please update the comment to include CUDA as well?

Comment (aelovikov-intel) [inline SYCL/Basic/barrier_order.cpp]:
```suggestion
// Temporarily disabled on HIP, CUDA and L0 due to sporadic failures.
```

**Review by sergey-semenov — APPROVED** [review]

**Comment by aelovikov-intel** [comment]
The failure

SYCL :: Plugin/level_zero_queue_priority.cpp

seems to be caused by the fact that pre-commit CI in llvm-test-suite uses nightly compiler and was broken for a day because the test requires corresponding change in intel/llvm.

@intel/llvm-gatekeepers , can this be merged in?

**Comment by pvchupin** [comment]
Thanks!

**Pull request merged**